src/Microsoft.Health.Fhir.Azure/IntegrationDataStore/AzureAccessTokenClientInitializerV2.cs

```diff
@@ -0,0 +1,75 @@
+﻿// -------------------------------------------------------------------------------------------------
```

Comment:
This file and the AzureConnectionStringClientInitializerV2 are identical to the versions used for Export except for the error types they throw. Would it be possible to make the existing versions compatible with both export and import?

Reply:
The current AzureConnectionStringClientInitialize load settings from ExportJobConfiguration. To make sure not impact current features on PaaS, we would like to add new config and ready on PaaS first, then migrate to new configuration.